server.js

```diff
@@ -1,28 +1,46 @@
 const express = require("express");
 const cors = require("cors");
-
-const { handleSignin } = require("./controllers/signin");
-const { handleSignout } = require("./controllers/signout");
-const { requireAuth } = require("./controllers/authorization");
-const { getLayout } = require("./controllers/drumLayout");
+const morgan = require("morgan");
+require("dotenv").config();
+
+const ErrorResponse = require("./utils/errorResponse");
+const errorHandler = require("./middleware/error");
+
+// Route files
+const routes = require("./routes");
+
+process.on("uncaughtException", err => {
+  // eslint-disable-next-line no-console
+  console.log("UNCAUGHT EXCEPTION! Shutting down...".red.bold);
+  // eslint-disable-next-line no-console
+  console.log(err.name, err.message);
+  // exit process
+  process.exit(1);
+});
 
 const app = express();
 
+// Dev logging middleware
+if (process.env.NODE_ENV === "development") {
+  app.use(morgan("dev"));
+}
+
 // Enable CORS
 app.use(cors());
 
 // Body parser
 app.use(express.json());
 
-app.get("/", (req, res) => {
-  res.send("It's Working!");
+// Mount routers
+app.use("/", routes);
+
+app.all("*", (req, res, next) => {
+  next(new ErrorResponse(`Can't find ${req.originalUrl} on this server!`, 404));
 });
 
-app.post("/signin", handleSignin);
-app.post("/signout", handleSignout);
-app.get("/drumlayout", requireAuth, getLayout);
+app.use(errorHandler);
 
-const PORT = process.env.PORT || 3000;
```

Comment:
- [ ] Let's switch this to just plan old `process.env.PORT` so that this will work in different future environments (dev vs prod)

Reply:
Done